subsys/bluetooth/audio/ascs.c

```diff
@@ -68,10 +70,32 @@ K_MEM_SLAB_DEFINE(ase_slab, sizeof(struct bt_ascs_ase),
 		  CONFIG_BT_ASCS_MAX_ACTIVE_ASES,
 		  __alignof__(struct bt_ascs_ase));
 
+#define MAX_CODEC_CONFIG \
+	MIN(UINT8_MAX, \
+	    CONFIG_BT_CODEC_MAX_DATA_COUNT * CONFIG_BT_CODEC_MAX_DATA_LEN)
+#define MAX_METADATA \
+	MIN(UINT8_MAX, \
+	    CONFIG_BT_CODEC_MAX_METADATA_COUNT * CONFIG_BT_CODEC_MAX_DATA_LEN)
+
+/* Minimum state size when in the codec configured state */
+#define MIN_CONFIG_STATE_SIZE (1 + 1 + 1 + 2 + 3 + 3 + 3 + 3 + 5 + 1)
```

Comment:
Seems you forgot about ASE_ID and ASE_State

```suggestion
#define MIN_CONFIG_STATE_SIZE (1 + 1 + 1 + 1 + 1 + 2 + 3 + 3 + 3 + 3 + 5 + 1)
```

ASE_ID (1) + ASE_State (1) + Framing (1) + Preferred_PHY (1) + Preferred_Retransmission_Number (1) + Max_Transport_Latency (2) + Presentation_Delay_Min (3) + Presentation_Delay_Max (3) + Preferred_Presentation_Delay_Min (3) + Preferred_Presentation_Delay_Max (3) + Codec_ID (5) + Codec_Specific_Configuration_Length (1)